plugins/resources/unixfilesystem/src/irods_is_in_host_list.cpp

```diff
@@ -0,0 +1,52 @@
+#include "irods/private/irods_is_in_host_list.hpp"
+
+#include "irods/filesystem/path.hpp"
```

Comment:
This header seems like it can be removed as well since nothing appears to use it.

Reply:
done